package.json

```diff
@@ -4,17 +4,15 @@
   "description": "browserify v2 plugin for jade with sourcemaps support",
   "main": "index.js",
   "dependencies": {
-    "browserify-transform-tools": "~1.5.1",
-    "convert-source-map": "~1.1.3",
-    "source-map": "~0.5.3",
+    "browserify-transform-tools": "~1.6.0",
+    "convert-source-map": "~1.2.0",
+    "jade": "~1.11.0",
```

Comment:
I actually tried to use `pug` and tried to replace all `jade` requires inside `index.js` with latest pug release but it doesn't work. Looks like migration on pug a more complex task than simply replace `jade` on `pug`.